src/test/java/org/testeditor/fixture/host/integrationtest/HostDriverIntegrationTest.java

```diff
@@ -121,4 +122,44 @@ public void typeAtTest() {
         hostDriverFixture.disconnect();
     }
 
+    @Test
+    public void newTransactionTest() {
+        assumeTrue();
```

Comment:
assumeTrue should be something like assumeWindowsAndS3270Accessible or something like that.